openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -32,6 +39,30 @@ def __init__(self, *args, **kwargs):
         super(CreateReview, self).__init__(*args, **kwargs)
         data = OrderedDict(**self.data)
 
+        project_name = get_current_project_name()
+        profiles = get_project_settings(
+            project_name
+        )["maya"]["publish"]["ExtractPlayblast"]["profiles"]
```

Comment:
What happens if `get_project_settings` is called but there are no saved settings with the new settings schema, does it then also already have the `profiles` entry?